decoder/body_extensions_test.go

```diff
@@ -533,7 +536,7 @@ func TestCompletionAtPos_BodySchema_Extensions(t *testing.T) {
 							},
 						},
 						NewText: "for_each",
-						Snippet: "for_each = ${1:1}",
+						Snippet: "for_each {\n ${1}\n}",
```

Comment:
I think the original snippet was correct - `for_each` is not a block, but an attribute, so there's no need for the curly braces and newlines.

Reply:
I was following the first example in https://www.terraform.io/language/meta-arguments/for_each#basic-syntax. I missed the quals sign, but thought since it shows map first that is the likely first choice to use